packages/db-authorization/test/endpoint.test.js

```diff
@@ -0,0 +1,644 @@
+'use strict'
+
+const { test } = require('tap')
+const fastify = require('fastify')
+const core = require('@platformatic/db-core')
+const { connInfo } = require('./helper')
+const auth = require('..')
+const restPlugins = require('../fixtures/plugin')
+
+test('Users cannot make a requests to /page endpoint if they are not allowed by endpoind authorization rules (REST)', async ({ pass, teardown, same, equal }) => {
+  const app = fastify()
+
+  app.register(core, {
+    ...connInfo
+  })
+
+  app.register(auth, {
+    jwt: {
+      secret: 'supersecret'
+    },
+    roleKey: 'X-PLATFORMATIC-ROLE',
+    anonymousRole: 'anonymous',
+    rules: [{
+      role: 'user',
```

Comment:
I don't understand what this rule means. find, save, and delete should be related to some entity.

Reply:
I am not a big fan of this either. It was implemented like that to look like the rules provided for entities